Add db-plan-enabled recommendation (#588)
PostgREST/postgrest-docs: docs/configuration.rst

  When this is set to :code:`true`, the execution plan of a request can be retrieved by using the :code:`Accept: application/vnd.pgrst.plan` header. See :ref:`explain_plan`.

>   It's recommended to only use this on testing environments and not in production as it reveals internal database details.
>   However if you choose to use it in production you can use a :ref:`db-pre-request` to filter the requests that can use this feature.

Reviewer: ```suggestion
  It's recommended to use this in testing environments only since it reveals internal database details.
  However, if you choose to use it in production you can implement a :ref:`db-pre-request` to filter the requests that can use this feature.
```

I suggested "implement" but any other word can be to replace "use", because it's repeated a lot.
